python/ray/tune/trainable/trainable.py

```diff
@@ -487,7 +490,8 @@ def save(self, checkpoint_dir: Optional[str] = None) -> str:
         TrainableUtil.write_metadata(checkpoint_dir, metadata)
 
         # Maybe sync to cloud
-        self._maybe_save_to_cloud(checkpoint_dir)
+        if not prevent_upload:
```

Comment:
How about syncing to driver?
I recall seeing these tmp folders on driver...

Reply:
As discussed, out of scope for this PR :-)

Reply:
(I'll address that in a different PR - this PR actually makes PBT fail while syncing to driver is not great but at least the experiment will continue to run).